test_requirements/requirements_base.txt

```diff
@@ -26,4 +26,4 @@ sphinx
 sphinxcontrib-spelling
 pyflakes
 pyenchant
-
+mock>=1.3.0,<2.0.0
```

Comment:
I respectfully reject the use of mock in the django CMS testsuite
Especially for everything using requests we definitely need **real** object to have useful tests